README.md

```diff
@@ -15,49 +15,70 @@
 import taichi as ti
 ```
 
-- [Getting Started](#getting-started)
-  - [Installation](#installation)
-  - [Your first Taichi program](#your-first-taichi-program)
-- [Documentation](#documentation)
-- [Contacts](#contacts)
-- [Contributing](#contributing)
-- [Resources](#resources)
-  - [Demos](#demos)
-  - [Lectures & talks](#lectures--talks)
-
-**Taichi (太极)** is an open-source, imperative, parallel programming language for high-performance numerical computation. It is embedded in Python and uses just-in-time (JIT) compiler frameworks (e.g. LLVM) to offload compute-intensive Python code to the native GPU or CPU instructions.
+# What is Taichi?
 
-Advantages of Taichi:
+Taichi is an open-source, imperative, parallel programming language for high-performance numerical computation. It is embedded in Python and uses just-in-time (JIT) compiler frameworks, for example LLVM, to offload the compute-intensive Python code to the native GPU or CPU instructions.
 
-- Built around Python: Taichi shares almost the same syntax with Python, allowing you to write algorithms with minimal language barrier. It is also well integrated into the Python ecosystem, such as NumPy and PyTorch.
-- Flexibility: Taichi provides a set of generic data containers known as *SNode* (/ˈsnoʊd/), an effective mechanism for composing hierarchical, multi-dimensional fields. This can cover many use patterns in numerical simulation (e.g. [spatially sparse computing](https://docs.taichi.graphics/lang/articles/advanced/sparse)).
-- Performance: Through the `@ti.kernel` decorator, Taichi's JIT compiler automatically compiles your Python functions into efficient GPU or CPU machine code for parallel execution.
-- Portability: Write your code once and run it everywhere. Currently, Taichi supports most mainstream GPU APIs, such as CUDA and Vulkan.
-- ... and many more features! A cross-platform, Vulkan-based 3D visualizer, [differentiable programming](https://docs.taichi.graphics/lang/articles/advanced/differentiable_programming),  [quantized computation](https://github.com/taichi-dev/quantaichi) (experimental), etc.
-
-# Getting Started
+<a href="https://github.com/taichi-dev/taichi/blob/master/python/taichi/examples/simulation/fractal.py#L1-L31"> <img src="https://github.com/taichi-dev/public_files/raw/master/taichi/fractal_code.png" height="270px"></a>  <img src="https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/fractal_small.gif" height="270px">
```

Comment:
Previously I removed the screenshot of the code and used code block instead, because I believe that was easier for users to directly copy/paste into their editor and give it a shot. The new change (or I should say the one before my change) would make fractal show up twice in README, once at the headline, the other at the Hello World link. (That somehow doesn't feel right to me)

Not sure about your opinions, tho?